UI/src/locales/ar_EG.json

```diff
@@ -0,0 +1,1817 @@
+{
+    "30": "30",
```

Comment:
You converted the entire translation lexicon, but we have only a very small amount of translation on the client side as of yet. Genuine question: should we only send the bits for which we have actual translations? (Obviously, we can only present the english version for anything we didn't translate yet, so maybe there's no reason to even *send* it? also, for any keys that aren't being used on the client for translation today, maybe we need not send those either? That'd be more efficient, but I have no idea about the downsides...)